lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -934,14 +935,171 @@ module.exports = class MethodMappings {
         ///////////////////////////////////////////////////////////////////////////
 
         async registerAuth(username, password) {
-          const pageCdpConnection = await this.driver.createCDPConnection('page');
-          await this.driver.register(username, password, pageCdpConnection);
+          const cdpConnection = await cdp.getConnection(this.driver);
+          await this.driver.register(username, password, cdpConnection);
 
           return {
             value: null
           };
+        },
+
+        async startLogsCapture(userCallback) {
+          const cdpConnection = await cdp.getConnection(this.driver);
+          await this.driver.onLogEvent(cdpConnection, userCallback);
+
+          return {
+            value: null
+          };
+        },
+
+        async catchJsExceptions(userCallback) {
+          const cdpConnection = await cdp.getConnection(this.driver);
+          await this.driver.onLogException(cdpConnection, userCallback);
+
+          return {
+            value: null
+          };
+        },
+
+        ///////////////////////////////////////////////////////////////////////////
+        // CDP commands
+        ///////////////////////////////////////////////////////////////////////////
+
+        async setGeolocation(coordinates) {
+          const cdpConnection = await cdp.getConnection(this.driver);
+
+          await cdpConnection.execute(
+            'Emulation.setGeolocationOverride',
+            coordinates
+          );
+
+          return {
+            value: null
+          };
+        },
+
+        async clearGeolocation() {
```

Comment:
I think we should include a command to `clearGeoLocation` as well

Reply:
I asked about this over Slack but @AutomatedTester [suggested](https://browserstack.slack.com/archives/C027A82KQC9/p1652186454786369?thread_ts=1652180780.347539&cid=C027A82KQC9) against it.